packages/grid/_modules_/grid/components/containers/GridRootStyles.ts

```diff
@@ -102,27 +102,34 @@ export const useStyles = makeStyles(
           display: 'flex',
           alignItems: 'center',
         },
+        '& .MuiDataGrid-colCell:not(.MuiDataGrid-colCellSorted) .MuiDataGrid-sortIcon': {
+          opacity: 0,
+          transition: theme.transitions.create(['opacity'], {
+            duration: theme.transitions.duration.shorter,
+          }),
+        },
+        '& .MuiDataGrid-colCell:not(.MuiDataGrid-colCellSorted):hover .MuiDataGrid-sortIcon': {
+          opacity: 0.5,
+        },
         '& .MuiDataGrid-colCellTitleContainer': {
-          textOverflow: 'ellipsis',
-          overflow: 'hidden',
-          whiteSpace: 'nowrap',
-          display: 'inline-flex',
+          display: 'flex',
+          alignItems: 'center',
+          minWidth: 0,
           flex: 1,
-        },
-        '& .MuiDataGrid-colCellNumeric .MuiDataGrid-iconButtonContainer': {
-          paddingRight: 5,
+          whiteSpace: 'nowrap',
         },
         '& .MuiDataGrid-colCellSortable': {
           cursor: 'pointer',
         },
-        '& .MuiDataGrid-sortIcon': {
-          fontSize: 18,
```

Comment:
It looks like we should add this style back. The icon should be 18px, not 20px wide.

Reply:
Well depends on what you read. it feels like 20px in https://material.io/components/data-tables#specs. In https://material-ui.com/components/tables/ we use 18px. On https://search.google.com/, it's 16px:

<img width="244" alt="Screenshot 2021-05-02 at 21 15 21" src="https://user-images.githubusercontent.com/3165635/116824633-8bd92600-ab8b-11eb-9c16-1013a120ac3b.png">

Reply:
I changed to `font-size: inherit` to not have a specific size for just one icon. They appear to have the same dimensions.

![image](https://user-images.githubusercontent.com/42154031/116894109-119ab700-ac08-11eb-83fa-53d1a906c0ea.png)

Reply:
I see, we rely on IconButton that has font-size 18px on size small. Ok looks good.